pep_sphinx_extensions/pep_processor/transforms/pep_headers.py

```diff
@@ -7,7 +7,42 @@
 
 from pep_sphinx_extensions.pep_processor.transforms import pep_zero
 from pep_sphinx_extensions.pep_processor.transforms.pep_zero import _mask_email
+from pep_sphinx_extensions.pep_zero_generator.constants import (
+    SPECIAL_STATUSES,
+    STATUS_ACCEPTED,
+    STATUS_ACTIVE,
+    STATUS_DEFERRED,
+    STATUS_DRAFT,
+    STATUS_FINAL,
+    STATUS_PROVISIONAL,
+    STATUS_REJECTED,
+    STATUS_SUPERSEDED,
+    STATUS_WITHDRAWN,
+    TYPE_INFO,
+    TYPE_PROCESS,
+    TYPE_STANDARDS,
+)
+
+ABBREVIATED_STATUSES = {
+    STATUS_DRAFT: "Proposal under active discussion and revision",
+    STATUS_DEFERRED: "Inactive draft that may be taken up again at a later time",
+    STATUS_ACCEPTED: "Normative proposal accepted for implementation",
+    STATUS_ACTIVE: "Currently valid/in-use guidance or process",
```

Comment:
```suggestion
    STATUS_ACTIVE: "Currently valid or in-use guidance or process",
```

Reply:
Well, its not necessarily (exclusive) "or" though; the word "or" at least to me carries an implication of exclusivity and a more tightly defined relationship between the two qualities than a slash, which is deliberately more broad. Furthermore, it results in a double "or", making the _overall grammar of the sentence_ more ambiguous and harder to parse at a glance, whereas the `/` spacing establishes a clearer syntactic hierarchy.

What about the following, which avoids the slash while being easier to read and clearer which adjectives apply to which words, as well as clearer on the distinction between the types of PEPs that Active vs. Accepted generally apply to (to reduce the ambiguity in the key between their identical characters).

```suggestion
    STATUS_ACTIVE: "Currently valid informational guidance, or an in-use process",
```